boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0.yaml

```diff
@@ -16,4 +16,5 @@ toolchain:
   - xtools
 supported:
   - gpio
+  - lpadc
```

Comment:
This should be `adc`, not the name of the device IP

Reply:
Thanks for your reminder, I will fix it.